packages/tsconfig.json

```diff
@@ -8,7 +8,6 @@
     "emitDecoratorMetadata": true,
     "experimentalDecorators": true,
     "target": "es6",
-    "sourceMap": true,
```

Comment:
We will want the `sourceMap` so we can properly debug ts files

Reply:
This is was causing a errors in the terminal in react so it was quick fix. It doesnt break the app though so i can add it back and we can write up a ticket for it